pypeit/images/pypeitimage.py

```diff
@@ -50,20 +50,34 @@ class PypeItImage(datamodel.DataContainer):
             Master key, only for Master frames
 
     """
-    version = '1.0.1'
+    version = '1.0.2'
```

Comment:
given how much is changing, I suggest
you go with `1.1.0`

Reply:
Good call.  Changed.